locale/fr/get-involved/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Comment s'impliquer ?
+layout: contribute.hbs
+---
+
+# Nous encourageons toutes les formes de contributions de la part de la communauté 
```

Comment:
Globally, this paragraph does not feel very idiomatic in French to me.

Reply:
delete 'les' ?

Reply:
I'd say if you want to delete "les", go singular. "Nous encourageons toute forme de contribution venant de la communauté" ?

Reply:
+1